src/AdminAcceptanceTests.Actions/Pages/EditOrganisation.cs

```diff
@@ -0,0 +1,37 @@
+﻿using AdminAcceptanceTests.Actions.Utils;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AdminAcceptanceTests.Actions.Pages
+{
+    public sealed class EditOrganisation : PageAction
+    {
+        public EditOrganisation(IWebDriver driver) : base(driver)
+        {
+
+        }
+
+        public void PageDisplayed()
+        {
+            Wait.Until(d => d.FindElements(Pages.EditOrganisation.PageTitle).Count > 0);
+            Wait.Until(d => d.FindElement(Pages.EditOrganisation.PageTitle).Displayed);
```

Comment:
By definition, the page title would be displayed if the driver can find more than 0 Page titles, surely?

Reply:
Depends on how 'Displayed' works as it could be hidden, but generally yes. I thought this might be a nice way to make sure it waits by having 2 of them.